src/_imaging.c

```diff
@@ -506,7 +506,7 @@ getink(PyObject* color, Imaging im, char* ink)
 #else
                 if (PyInt_Check(color) || PyLong_Check(color)) {
                         if (PyInt_Check(color))
-                                r = PyInt_AS_LONG(color);
+                                r = (int)PyInt_AS_LONG(color);
```

Comment:
The comment near r declaration mentions that r could be up to 0xFFFFFFFF, which is overflow for `int`.
r is declared as PY_LONG_LONG, PyInt_AS_LONG really returns `PyIntObjectob_ival.ob_ival`, which is declared as long.

Is there really was a warning in this place?

Reply:
I actually can no longer find these warnings on Travis. So, feel free to pick up this PR if you want, but for my part, without a problem to solve, I am willing to consider this resolved.